Formula/opensearch-dashboards.rb

```diff
@@ -4,19 +4,20 @@ class OpensearchDashboards < Formula
   desc "Open source visualization dashboards for OpenSearch"
   homepage "https://opensearch.org/docs/dashboards/index/"
   url "https://github.com/opensearch-project/OpenSearch-Dashboards.git",
-      tag:      "1.3.2",
-      revision: "6aa55aee1acd98035c714a46c8508a5b5ecabfd5"
+      tag:      "2.0.0",
+      revision: "3d6dd638d021f383a4c6ab750c83a1d30d3787b3"
   license "Apache-2.0"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "849889b4333f56fdca11b839fa6c7dfbebe70b6cca4dafdb624f1d2c4154b2e6"
   end
 
   depends_on "yarn" => :build
-  depends_on "node@10" # Switch to `node` after https://github.com/opensearch-project/OpenSearch-Dashboards/issues/406
+  depends_on arch: :x86_64
+  depends_on "node@14"
```

Comment:
We can still mention the same issue for switching to node 16/18.

Reply:
Added comments in #102427